platform/ios/os_ios.mm

```diff
@@ -181,7 +177,10 @@ void register_dynamic_symbol(char *name, void *address) {
 }
 
 void OS_IOS::start() {
-	Main::start();
+	if (Main::start()) {
+		main_loop->initialize();
+		
```

Comment:
The stray line before the closing bracket fails the style check. Note that you can run clang-format automatically by setting up the pre-commit hooks with `pip install pre-commit && pre-commit install` in the repo.